src/routes/auth/logout.ts

```diff
@@ -14,15 +14,19 @@ interface HasuraData {
   auth_refresh_tokens: { account: AccountData }[]
 }
 
-async function logout({ body, cookies, signedCookies }: Request, res: Response): Promise<unknown> {
+async function logout({ body, cookies, signedCookies, query }: Request, res: Response): Promise<unknown> {
```

Comment:
Doesn't the auth middleware already put the refresh_token, whether it's on the query string or in a cookie, in the request? So, this could be:

```
async function logout({ body, cookies, signedCookies, refresh_token }: RequestExtended
```

Then you don't need to change anything else.

Reply:
Correct